ql/src/java/org/apache/hadoop/hive/ql/exec/DataSketchesFunctions.java

```diff
@@ -53,169 +72,256 @@
   private static final String SKETCH_TO_VARIANCES = "variances";
   private static final String SKETCH_TO_PERCENTILE = "percentile";
 
-  private final Registry system;
+  private List<SketchDescriptor> sketchClasses;
+
+  public DataSketchesFunctions() {
```

Comment:
This could be private to prevent instantiation. If sketch classes are loaded statically, it can be probably be empty.